src/plugins/taskbar-progress/index.ts

```
@@ -0,0 +1,166 @@
+    baseElapsedSeconds = songInfo.elapsedSeconds ?? 0;
+    progressInterval = setInterval(() => {
+      if (
+        lastSongInfo &&
+        !lastSongInfo.isPaused &&
+        baseElapsedSeconds !== null &&
+        intervalStart !== null
+      ) {
+        const timeDelta = (performance.now() - intervalStart) / 1000;
+        const elapsedSeconds = Math.floor(
+          baseElapsedSeconds + timeDelta,
+        );
```

ESLint

[error] Replace `⏎··········baseElapsedSeconds·+·timeDelta,⏎········` with `baseElapsedSeconds·+·timeDelta`

(prettier/prettier)